src/ontology/uberon-edit.obo

```diff
@@ -84444,6 +84444,7 @@ name: lower leg skin
 def: "A zone of skin that is part of a lower leg [Automatically generated definition]." [OBOL:automatic]
 synonym: "hind limb middle limb segment skin" EXACT [OBOL:automatic]
 synonym: "hind limb zeugopod skin" EXACT [OBOL:automatic]
+synonym: "hindlimb zeugopod skin" EXACT [PMID:11943340]
```

Comment:
Thank you Paula, I can't see the usage of this synonym in the reference you have provided. Could it be the wrong PMID?

Reply:
@aleixpuigb thank you Aleix for reviewing. Yes, you are right. We just realized that hindlimb zeugopod skin was missing while integrating in Cellosauurs cell lines described in PMID:11943340 (but the term is not described in that paper). I will remove this reference but I do not have any other to add. Should I add my ORCID or leave it empty? Sorry for the confusion

Reply:
I think adding your ORCID is enough in this case, thank you!